subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/fixture/PerformanceTestJvmOptions.groovy

```diff
@@ -22,7 +22,7 @@ import org.gradle.api.JavaVersion
 
 @CompileStatic
 class PerformanceTestJvmOptions {
```

Comment:
It is still used by some non-Java tests (e.g. `NativeBuildPerformanceTest`). I would leave it in until we refactored those as well.